Aquality.Selenium.Core/src/Aquality.Selenium.Core/Applications/AqualityServices.cs

```diff
@@ -4,13 +4,13 @@
 
 namespace Aquality.Selenium.Core.Applications
 {
-    public abstract class ApplicationManager<TApplication>
+    public abstract class AqualityServices<TApplication>
         where TApplication : class, IApplication
     {
         private static readonly ThreadLocal<TApplication> AppContainer = new ThreadLocal<TApplication>();
         private static readonly ThreadLocal<IServiceProvider> ServiceProviderContainer = new ThreadLocal<IServiceProvider>();
 
-        protected ApplicationManager()
+        protected AqualityServices()
         {
         }
 
```

Comment:
I'm thinking about making the method IsApplicationStarted() also protected, what do you think about it? Does it make any sense?

Reply:
make sence, updated